packages/dev-app/src/views/HorizontalNavView.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import {
+	HorizontalNav,
+	HorizontalNavLink
+} from "@nice-digital/nds-horizontal-nav";
+
+const links = [
+	{
+		title: "Form Group",
+		destination: "/formgroup",
+		isCurrent: false,
+		elementType: Link
+	},
+	{
+		title: "Page Header",
+		destination: "/pageheader",
+		isCurrent: true,
+		elementType: Link
+	},
+	{
+		title: "Stacked Nav",
+		destination: "/stackednav",
+		isCurrent: false,
+		elementType: Link
+	},
+	{
+		title: "Alert",
+		destination: "/alert",
+		isCurrent: false,
+		elementType: Link
+	},
+	{
+		title: "Grid",
+		destination: "/grid",
+		isCurrent: false,
+		elementType: Link
+	}
+];
+
+const linksRealistic = [
```

Comment:
Where do you use these?

Reply:
This is just where I was playing with more real-life variations when stress-testing. The dev app is a playground of sorts. So yes that's a currently unused variable.